core/templates/domain/skill/concept-card-backend-api.service.spec.ts

```diff
@@ -54,12 +49,12 @@ describe('Concept card backend API service', function() {
     };
     var example2 = {
       question: {
-        html: 'worked example question 1',
-        content_id: 'worked_example_q_1'
+        html: 'worked example question 2',
```

Comment:
Why this example is being changed?

Reply:
example 2 is identical to example  1. This must have been an oversight when the file was built out so I corrected the same.

This does not impact the tests

Reply:
Sounds good.